jinahub/rankers/t.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+from jina import Flow, Document
+from PIL import Image
+
+f = Flow().add(uses='jinahub+docker://CLIPImageEncoder')
+
+def print_result(resp):
+    doc = resp.docs[0]
+    print(f'Embedding image to {doc.embedding.shape[0]}-dimensional vector')
+
+with f:
+    doc = Document(blob=np.asarray(Image.open('myimage.png')))
+    f.post(on='/foo', inputs=doc, on_done=print_result)
+
```

Comment:
I think there's an issue with encoding
I receive this:
```
Traceback (most recent call last):
  File "/home/ec2-user/jina/test-executors/venv/lib/python3.7/site-packages/jina/clients/helper.py", line 60, in _arg_wrapper
    return func(*args, **kwargs)
  File "test_clip_image.py", line 9, in print_result
    print(f'Embedding image to {doc.embedding.shape[0]}-dimensional vector')
AttributeError: 'NoneType' object has no attribute 'shape'
```
can you check it ?

Reply:
Works perfectly for me. Can you check that you really have an image in your working directory?

Reply:
Actually, was able to reproduce. For some images it gives 4 channels, which needs to be converted to 3 channels.